packages/commerce-sdk-react/src/hooks/useMutation.ts

```diff
@@ -67,12 +68,22 @@ export const useCustomMutation = (
     apiOptions: OptionalCustomEndpointClientConfig,
     mutationOptions?: UseMutationOptions
 ) => {
-    const config = useConfig()
     const auth = useAuthContext()
+    const config = useConfig()
+    const clientHeaders = config.headers || {}
+    const currentClientConfig = {
+        parameters: {
+            clientId: config.clientId,
+            siteId: config.siteId,
+            organizationId: config.organizationId,
+            shortCode: config.organizationId
```

Comment:
why is shortCode assigned to a org Id value?

Reply:
That looks like a typo. Good catch. @joeluong-sfcc ?